### PR TITLE
fix: integration tests timeout in kind-client test

### PR DESCRIPTION
**Pull request by Ivo-Yankov**
## Description

This pull request changes the following:

* TBD

### Related Issues

* Closes #
* Related to #

### Pull request (PR) checklist

* \[ ] This PR added tests (unit, integration, and/or end-to-end)
* \[ ] This PR updated documentation
* \[ ] This PR added no TODOs or commented out code
* \[ ] This PR has no breaking changes
* \[ ] Any technical debt has been documented as a separate issue and linked to this PR
* \[ ] Any `package.json` changes have been explained to and approved by a repository manager
* \[ ] All related issues have been linked to this PR
* \[ ] All changes in this PR are included in the description
* \[ ] When this PR merges the commits will be squashed and the title will be used as the commit message, the 'commit message guidelines' below have been followed

### Testing

* \[ ] This PR added unit tests
* \[ ] This PR added integration/end-to-end tests
* \[ ] These changes required manual testing that is documented below
* \[ ] Anything not tested is documented

The following manual testing was done:

* TBD

The following was not tested:

* TBD

<details>
<summary>
Commit message guidelines
</summary>
We use 'Conventional Commits' to ensure that our commit messages are easy to read, follow a consistent format, and for automated release note generation. Please follow the guidelines below when writing your commit messages:

1. BREAKING CHANGE: a commit that has a footer BREAKING CHANGE:, or appends a ! after the type/scope, introduces a breaking API change (correlating with MAJOR in Semantic Versioning). A BREAKING CHANGE can be part of commits of any type.  NOTE: currently breaking changes will only bump the MAJOR version.
2. The title is prefixed with one of the following:

| Prefix    | Description                                         | Semantic Version Update | Captured in Release Notes |
|-----------|-----------------------------------------------------|-------------------------|---------------------------|
| feat:     | a new feature                                       | MINOR                   | Yes                       |
| fix:      | a bug fix                                           | PATCH                   | Yes                       |
| perf:     | performance                                         | PATCH                   | Yes                       |
| refactor: | code change that isn't feature or fix               | none                    | No                        |
| test:     | adding missing tests                                | none                    | No                        |
| docs:     | changes to documentation                            | none                    | Yes                        |
| build:    | changes to build process                            | none                    | No                        |
| ci:       | changes to CI configuration                         | none                    | No                        |
| style:    | formatting, missing semi-colons, etc                | none                    | No                        |
| chore:    | updating grunt tasks etc; no production code change | none                    | No                        |

</details>

### CODE REVIEW

File: test/e2e/integration/core/kind/kind-client.test.ts [hunk 1]
```diff
@@ -215,4 +218,4 @@ describe('KindClient Integration Tests', (): void => {
     const deletedCluster: KindCluster = clusters.find((c: KindCluster): boolean => c.name === testClusterName);
     expect(deletedCluster).to.be.undefined;
   });
-}).timeout(Duration.ofMinutes(1).toMillis());
```
Comment (jeromy-cannon) [inline test/e2e/integration/core/kind/kind-client.test.ts:218]: If this doesn't work correctly, then we probably need to back out my other describe timeout changes that I did in the same PR.

Comment (Ivo-Yankov) [reply test/e2e/integration/core/kind/kind-client.test.ts:218]: Supposedly, this is only an issue in this test suite, because some of the internal `it(...)` statements set a longer timeout. The `timeout` at the end of the `describe` overrides the internal `it` timeouts, whereas if it were declared as it was previously - as `this.timeout(...)`, then it is the other way around. I'll double-check if there are any other instances where this may become a problem.

**Review by jeromy-cannon — APPROVED** [review]
